%YAML:1.0
rho: 1.3016839109671283e-01
support_vectors: !!opencv-matrix
   rows: 6
   cols: 1
   dt: d
   data: [ -4.15378308e-04, -2.76017381e-04, -2.70105258e-04, 1.94210625e-05, 1.31084889e-05, 1.40484963e-05 ]
